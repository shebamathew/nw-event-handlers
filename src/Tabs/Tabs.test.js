import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer'; 
import Tabs from './Tabs';

describe(`Tabs Component`, () => {
  const tabsProp = [{name:'', content:''}];
  it('renders without errors', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Tabs />, div);
    ReactDOM.unmountComponentAtNode(div);
  })
  it('renders the first tab by default', () => {
    const tree = renderer.create(<Tabs tabs={tabsProp} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})