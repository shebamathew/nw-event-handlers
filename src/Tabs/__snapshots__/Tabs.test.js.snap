// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs Component renders the first tab by default 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    
  </button>
  <div
    className="content"
  >
    
  </div>
</div>
`;
